README.md

```diff
@@ -76,7 +76,7 @@ Get the cat model in a service
 import { Injectable } from "@nestjs/common";
 import { InjectModel } from "nestjs-typegoose";
 import { Cat } from "./cat.model";
-import { ModelType } from "@hasezoey/typegoose";
+import { ModelType } from "@typegoose/typegoose";
```

Comment:
```suggestion

```

Reply:
And for line 83. I don't know how to change. According to doc we have to use `getModelForClass`.

Reply:
I think that `getModelForClass` has been accounted for in `src/typegoose.providers.ts`.

Reply:
done